app/assets/javascripts/admin/api_flow_types.js

```diff
@@ -236,4 +236,19 @@ export type APITimeTrackingType = {
   tasktype_summary: string,
 };
 
+export type APIProjectProgressReportType = Array<{
```

Comment:
I would prefer type to be of a single instance. Then we have the flexibility to use them in arrays or single objects as need. Additionally, return types and variable declaration will inform you clearly that the are an array type.